src/blog/2020/001-dmn-js-8.md

```diff
@@ -0,0 +1,103 @@
+---
+
+title: DMN 1.3 Support in dmn-js
+layout: blogpost.hbs
+
+slug: 2020-dmn-js-8-0-0
+author: Philipp Fromme<https://github.com/philippfromme>
+
+published: 2020-02-28 12:00
+
+releases:
+  - 'dmn-js@8.0.0'
+
+---
+
+
+<p class="introduction">
+  We're excited to announce the latest release of our [DMN modeling toolkit](https://github.com/bpmn-io/dmn-js). The release supports DMN 1.3, the next version of the DMN standard. Facilitating the latest DMN features, we now export graphical information in a standardized format. Along with this release, we publish a migration path that allows you to open DMN 1.1 diagrams using the toolkit or batch migrate all your DMN 1.1 diagrams to DMN 1.3.

```

Comment:
> Along with this release, we publish a migration path [...]

Maybe change to: Along with this release, we __introduce__ a migration path [...]